src/PlugWallet/index.ts

```diff
@@ -237,12 +237,12 @@ class PlugWallet {
   };
 
   public removeToken = (tokenId: string) => {
-    if(!(tokenId in this.registeredTokens))
+    if(!(Object.keys(this.registeredTokens).includes(tokenId))) {
```

Comment:
This change should be the same, the operator in check into the keys

Reply:
it was failing that way